Dockerfile

```diff
@@ -1,27 +1,8 @@
-# oplog-dump worker
-FROM ubuntu:14.04
+FROM google/debian:wheezy
```

Comment:
it can't seem to find the image `Tag jessie not found in repository docker.io/google/debian` so i'm just going to stay with wheezy for now